### PR TITLE
Load json and yaml

### PR DESCRIPTION
**Pull request by ctb**
#74 again :)

- [x] Is it mergeable?
- [x] `make test` Did it pass the tests?
- [x] `make coverage` Is the new code covered?
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

**Comment by ctb** [comment]
Is anyone else working on this right now or shall I resolve?

**Comment by lgautier** [comment]
I am not working on it.

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ env:
 
 install:
 - pip install tox
+- pip install ijson
```
Comment (luizirber) [inline .travis.yml]: No need for this

File: sourmash_lib/signature.py [hunk 2]
```diff
@@ -149,16 +150,47 @@ def load_signatures(data, select_ksize=None, select_moltype=None,
...
+                if not select_moltype or \
+                     sig.estimator.is_molecule_type(select_moltype):
+                    yield sig
+    finally:
+        if is_fp:
+            data.close()
+
+
+def save_signatures(siglist, fp=None):
+    "Save multiple signatures into a YAML string (or into file handle 'fp')"
```
Comment (luizirber) [inline sourmash_lib/signature.py]: JSON, not YAML =]

**Pull request merged**